CRM/CiviAwards/Setup/RenameAwardTypeField.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+/**
+ * Renames `civiawards_award_type` option group to `civiawards_award_subtype`.
+ */
+class CRM_CiviAwards_Setup_RenameAwardTypeField {
```

Comment:
We don't need to have this as a setup. It should be in the upgrader file. A setup file is useful when it can be used by both the installer and the upgrader but in this case the installer uses an SQL file for creating the Award detail table and the column so this file is only useful to the upgrader. Not worth creating a separate class for as it will become redundant.